docs/installation.rst

```diff
@@ -2,16 +2,10 @@
 Installation
 ============
 
-Prerequisites
--------------
-* python V3.6 or later.
-* for interactive use, an ansi (vt100-compatible) terminal application.  On Windows 10, the 
-  separately-installable `Windows Terminal app <https://github.com/microsoft/terminal/releases>`_ 
-  is recommended.
-  
 Installation
 ------------
-Xonsh can be installed via package manager, via appimage or just run from a pre-built container:
+
+Xonsh can be installed via different ways:
```

Comment:
The original sentence here sounds better.

Reply:
The text just duplicate the list:

![Screenshot_20201009_022820](https://user-images.githubusercontent.com/1708680/95523382-28ad5980-09d7-11eb-9dfd-58f5958e83a5.png)

Reply:
Okay I replaced it to "Xonsh can be installed via different package managers from python specific to operation system package managers."